CHANGELOG.md

```diff
@@ -1,33 +0,0 @@
-### 0.1.2 (2015-11-25)
```

Comment:
Because the dependencies were all very outdated and won't transpile anymore correctly. There were a lot of dependencies that were like that. I wanted to have a clean start of sorts and the changelog is no major feature. But I'm happy to add it again (was the plan anyway)